Common Distributions-Distributions.ipynb

```diff
@@ -0,0 +1,717 @@
+{
```

Comment:
<p>As nu approaches +infinity, StudentT becomes Normal</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>Do we need to mention this? I think we can leave it out since were focusing on real world applications and uses, not statistical properties</p><p> </p>

Reply:
<p>I think people will ask what nu is, because you still have to fix or infer it</p>

Reply:
<p>Agreed, we don't need to say that it approaches the Normal but just with increasing nu the tails become smaller and smaller.</p>